www/apps/docs/content/experimental/index.md

```diff
@@ -0,0 +1,11 @@
+# Experimental Features
+
+This section of the documentation includes features that are currently experimental.
+
+As Medusa moves towards modularization, commerce concepts such as Products or Pricing will be moved to isolated commerce modules shipped as NPM packages. This significantly changes Medusa's architecture, making it more flexible for custom digital commerce applications.
+
+## Enabling Experimental Features
+
+Experimental Features are guarded in the Medusa backend by a feature flag.
+
+To use them, [enable the `medusa_v2` feature flag](../development/feature-flags/toggle.md) in your backend.
```

Comment:
For existing users, they need to run a migration after enabling the flag, should we add that in here?

Reply:
Good point! Will add it in